packages/ripple-nuxt-tide/lib/templates/axios.js

```diff
@@ -12,6 +12,30 @@ export default function ({ $axios, app, res }) {
     }
   })
 
+  $axios.onResponse(response => {
+    // Pass Drupal content page response section cache tag to Nuxt page response.
+    // Below content endpoints are hard coded as not need to be extensible for now.
+    const contentReqPatterns = [
+      '/api/v.*/node.*', 
```

Comment:
If it's not too hard can this pull from nuxt.config.json on the project level instead? Also, I think the menu api all might need to get added here?

Reply:
will see if I can add it as a custom config.